### DISCUSSION TITLE
can we dismiss the recommendation tabs for whatever reasons?

### DISCUSSION THREAD

**Discussion by kymtwyf** [community/community#18071] [Other Feature Feedback, Questions, & Ideas] [answered]
The feeds are cool to use. However, sometimes it may recommend something (user/repo/activity) but I would like to dismiss the tab if I am not interested in this. 

<img width="355" alt="image" src="https://user-images.githubusercontent.com/1797303/172525886-3519fc19-b15b-421e-9965-5cd11ea7e132.png">

**Comment by siminapasat** [comment, marked as answer]
Thanks for your feedback @kymtwyf ! We're actively working on this ☝️
